x/observer/types/chain_params_test.go

```diff
@@ -196,4 +197,23 @@ func (s *UpdateChainParamsSuite) Validate(params *types.ChainParams) {
 	copy.OutboundTxScheduleLookahead = 501
 	err = types.ValidateChainParams(&copy)
 	require.NotNil(s.T(), err)
+
```

Comment:
@ws4charlie I may think from git history that you worked on the tests in this file?
Why did we define a test suite for this?